src/test/fixtures/surveyA001_1.js

```diff
@@ -0,0 +1,227 @@
+export default {
+  surveyA001_1: {
+    survey_id: 'A001-1',
+    questions: [
+      {
+        question_index: 0,
+        question: '왜 해당 멘토를 선택하였는가?',
```

Comment:
I set this from 'Yoda Sprint Planning' sprint#4 and it wasn't English..T.T I updated it please check!